neurox/interpretation/probeless.py

```diff
@@ -0,0 +1,118 @@
+"""Module for Probeless method
+This module extracts neuron ranking for an attribute or for the entire property set .
```

Comment:
I think better to use the language "tag" or "label" instead of "attribute" and "property set" to maintain consistency in the toolkit; I'm sure we break this elsewhere but lets not add to the chaos :D